test/unit/general/deviceService-test.js

```diff
@@ -180,6 +210,7 @@ describe('NGSI-v2 - Device Service: utils', function () {
         async.series([iotAgentLib.clearAll, iotAgentLib.deactivate], done);
     });
 
+    //FIXME: this test will be removed if at the end /iot/services API (now Deprecated) is removed
```

Comment:
```suggestion
    // #FIXME1649: this test will be removed if at the end /iot/services API (now Deprecated) is removed
```

To use always the same stile for #1649 related comments...